plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8077,16 +8077,8 @@ private void testCreateTableWithCompressionCodec(Session baseSession, HiveStorag
                 .build();
         String tableName = "test_table_with_compression_" + compressionCodec;
         String createTableSql = format("CREATE TABLE %s WITH (format = '%s') AS TABLE tpch.tiny.nation", tableName, storageFormat);
-        if (storageFormat == HiveStorageFormat.PARQUET && compressionCodec == HiveCompressionCodec.LZ4) {
-            // TODO (https://github.com/trinodb/trino/issues/9142) Support LZ4 compression with native Parquet writer
```

Comment:
todo still valid.

Reply:
agreed (and todo link remains somewhere in the code)

Reply:
will restore in the test code